docusaurus.config.js

```diff
@@ -135,4 +135,12 @@ module.exports = {
       },
     ],
   ],
+  plugins: [
```

Comment:
I just removed this it shouldn't be in this diff, seems like a GitHub bug. Rebasing should fix that.